setup:
    prefix: 'j2s7s300_driver'
    model_filename: "jaco_dynamics"
    object_centers: {'HUMAN_CENTER': [-0.6,-0.55,0.0], 'LAPTOP_CENTER': [-0.7929,-0.1,0.0]}
    start: [104.2, 151.6, 183.8, 101.8, 224.2, 216.9, 310.8]
    T: 20.0
    timestep: 0.5

controller:
    # These settings have been tuned for PID controller.
    type: "pid"
    p_gain: 50.0
    i_gain: 0.0
    d_gain: 20.0
    epsilon: 0.10
    max_cmd: 40.0


